cmake/modules/FindCython.cmake

```diff
@@ -7,8 +7,6 @@ SET(Cython${PYTHON_VERSION}_FOUND FALSE)
 execute_process(
     COMMAND ${PYTHON${PYTHON_VERSION}_EXECUTABLE} -m cython --version
     RESULT_VARIABLE CYTHON_RESULT
-    OUTPUT_QUIET
```

Comment:
how about printing the stdout and stderr only `If (Cython${PYTHON_VERSION}_FOUND)`

Reply:
Upon success, as @tserong says, this only produces one line of output which, I would argue, provides useful information.

Upon failure, all the output is needed for debugging the failure.

Reply:
@smithfarm i think, no news is good news. just wanted to avoid unused output in success runs.  i can help on this if you want.

Reply:
@tchaikov Please go ahead.

Reply:
@smithfarm #22949